src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -88,12 +77,8 @@ const TableHeader: React.FC<TableHeaderProps> = ({
   );
 
   const tableTitle = tableOfUser?.title ?? "Table";
-  const tableCapacity = tableOfUser?.capacity;
-  const numberOfSeatsLeft = useMemo(() => {
-    if (!tableCapacity || !usersAtCurrentTable) return 0;
-
-    return tableCapacity - usersAtCurrentTable.length;
-  }, [tableCapacity, usersAtCurrentTable]);
```

Comment:
Is this not needed anymore? And if not, why?

Reply:
The design now shows the maximum available seats instead of the remaining amount of seats.

Reply:
This logic was calculating max seats - taken seats to get the `n` amount of seats remaining.

Reply:
Makes sense. Sounds good.